tika/tika/tika.py

```diff
@@ -41,6 +41,11 @@ def __init__(self, config: ConfigParser, plugin_opts: Optional[Dict]) -> None:
         elif config.has_option('options', 'tika_url'):
             self.tika_url = config.get('options', 'tika_url')
 
+        if plugin_opts and 'tika_save_text' in plugin_opts:
```

Comment:
I know this is being pedantic, but do you mind updating the variable to be `save_text`? Also, do you mind adding the configuration option to the `README` as well as the `tika.stoq` configuration file, along with a description of what it does?